src/Request/RegionRequest.php

```diff
@@ -28,7 +28,7 @@ class RegionRequest extends BaseApiRequest
     ];
 
     const SORTS = [
-        'name',
+        'id',
```

Comment:
it works because we changed the cacheKey to code?

Reply:
The sort order here isn't really useful. The regions get sorted by php in the new websites.

Reply:
okay, I see how it is fixed in the new website logic. We should fix the sorting because we can't use a local sorting every place where we use it in the future. Also, once we make it available for the customers they will complain about it.

Reply:
I totally agree, but we can't sort naturally in MySQL afaik.

Reply:
@wolfgangweintritt Können wir die Sort Methode hier überschreiben und einfach mit PHP nach Namen sortieren?

Reply:
yep

Reply:
Result: The effort is not worth the result. The political districts are sorted in the ReadModels and the websites.